src/BottomTabs.re

```diff
@@ -123,27 +123,50 @@ module Make = (M: {type params;}) => {
       ~tabBarButtonComponent: React.element=?,
       unit
     ) =>
-    options =
-    "";
+    options;
 
   type optionsProps = {
     navigation,
-    route: route(M.params),
+    route,
   };
 
   type optionsCallback = optionsProps => options;
 
-  type navigatorProps;
+  type navigatorProps = {
+    initialRouteName: option(string),
+    screenOptions: option(optionsCallback),
+    _lazy: option(bool),
+    tabBar: option(React.component(Js.t(bottomTabBarProps))),
+    tabBarOptions: option(bottomTabBarOptions),
+  };
+
+  type renderCallbackProp = {
+    navigation,
+    route,
+  };
 
-  type screenProps;
+  type screenProps('params) = {
+    name: string,
+    options: option(optionsCallback),
+    initialParams: option('params),
+    component:
+      option(
+        React.component({
+          .
+          "navigation": navigation,
+          "route": route,
```

Comment:
Can this be declared as a standard record?

Reply:
I don't know, I copied the way it was done on the Stack 😅
https://github.com/reason-react-native/reason-react-navigation/blob/a7d40f04e6785ed0c4525ad2b5f63a287282cf7f/src/Stack.re#L127
https://github.com/reason-react-native/reason-react-navigation/blob/a7d40f04e6785ed0c4525ad2b5f63a287282cf7f/src/Stack.re#L269-L282

Reply:
Ok so we could change that later in all places